radix-engine-tests/tests/blueprints/proof_creation/src/lib.rs

```diff
@@ -18,63 +18,69 @@ mod pc {
 
         pub fn create_proof_from_fungible_bucket() {
             let bucket = Self::create_fungible_bucket();
-            let proof = bucket.create_proof();
+            let proof = bucket.create_proof().skip_checking();
```

Comment:
Out of interest, why do we now need to skip checking on proofs created from buckets?

Reply:
Because if we have a bucket from function arguments and create a proof from it. We may forget to check the address.